src/pseudopeople/configuration.py

```diff
@@ -62,45 +67,49 @@ def get_configuration(user_configuration: Union[Path, str, Dict] = None) -> Conf
 
     # Instantiate the configuration file with baseline values
     baseline_dict = {}
+
+    # Loop through each form
     for form in FORMS:
-        if not form.is_implemented:
-            continue
-        baseline_dict[form.name] = {}
-        baseline_dict[form.name][Keys.ROW_NOISE] = {}
-        baseline_dict[form.name][Keys.COLUMN_NOISE] = {}
+        form_dict = {}
+        row_noise_dict = {}
+        column_dict = {}
+
+        # Loop through row noise types
         for row_noise in form.row_noise_types:
-            if not row_noise.is_implemented:
-                continue
+            row_noise_type_dict = {}
```

Comment:
It was kinda left up in the air and I decided to go ahead and get rid of that. But then instead of fully deleting stuff I just commented it out as a reminder/template for what's needed.